archey/logos/nobara.py

```diff
@@ -0,0 +1,29 @@
+"""Nobara logo"""
+
+from archey.colors import Colors
+
+COLORS = [Colors.WHITE_BRIGHT]
+
+# pylint: disable=line-too-long
+LOGO = [
+    """{c[0]}  .cONWMWNOl;cdkKXWWMMMMWWXKkdc'.         """,
```

Comment:
Hi @Snootiful ! Thanks for this very complete PR.

What about removing one (or both) leading whitespace(s) for this new logo ?

```suggestion
    """{c[0]} .cONWMWNOl;cdkKXWWMMMMWWXKkdc'.         """,
```

Thanks, bye 👋